### ISSUE TITLE
setStartRow() missing

### ISSUE THREAD

**Issue by hkirsman** [google/google-api-php-client#927]
In \Google_Service_Webmasters_SearchAnalyticsQueryRequest() there's setStartDate(), setEndDate(), setDimensions(), setRowLimit() but no setStartRow() or similar.

According to this https://developers.google.com/webmaster-tools/v3/how-tos/search_analytics#getting-more-than-5000-rows there's "startRow" option in the API.

**Comment by hkirsman** [comment]
Ok, it's available in apiclient-services so instead of (in composer.json):
```
{
    "require": {
        "google/apiclient": "^2.0.0@RC"
    }
}
```
I do this:
```
{
    "require": {
        "google/apiclient": "2.0.0-RC7",
        "google/apiclient-services": "@dev"
    }
}
```

**Comment by bshaffer** [comment]
Google/apiclient 2.0@RC pulls in apiclient-services. It is redundant to
include it in your composer.json

**Comment by hkirsman** [comment]
Pretty sure it didn't. But I'll recheck.

**Comment by bshaffer** [comment]
This was added in RC7, so it is possible you were previously on RC6 or
below and had not run "composer update" since RC7 was added.

**Comment by hkirsman** [comment]
If I remove it from composer.json then I get:
```
⇒  composer update
You are running composer with xdebug enabled. This has a major impact on runtime performance. See https://getcomposer.org/xdebug
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for google/apiclient 2.0.0-RC7 -> satisfiable by google/apiclient[v2.0.0-RC7].
    - google/apiclient v2.0.0-RC7 requires google/apiclient-services *@dev -> no matching package found.

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://getcomposer.org/doc/04-schema.md#minimum-stability> for more details.

Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.
```

I'm trying both:
```
{
    "require": {
        "google/apiclient": "2.0.0-RC7"
    }
}

```
and
```
{

    "require": {
        "google/apiclient": "^2.0.0@RC"
    }
}

```

But latter doesn't include the setStartRow() probably because it can't find apiclient-services.

**Comment by bshaffer** [comment]
You're right, composer was skipping RC7 in favor of RC6 unless the minimum stability flag was explicitly set.

I fixed this by tagging the `v0.1` release in client services. Try to update and see if that fixed it.

**Comment by hkirsman** [comment]
This works! Thank you very much!
